src/components/Album.jsx

```diff
@@ -6,6 +6,8 @@ export const Album = ({ album }) => {
 
   return (
     <div className="album-container">
+
+      ll
```

Comment:
I think this might have been added accidentally?

Reply:
Umm, yes, strange. I'll remove it